blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -227,6 +327,26 @@ exports.fetchLogsInjectStart = function fetchLogsInjectStart(url, start, onSucce
             }
         });
 };
+
+/**
+ * Determines if the provided object is a "run" type
+ */
+function isRun(obj) {
+    return obj._class === 'io.jenkins.blueocean.rest.impl.pipeline.PipelineRunImpl'
```

Comment:
I know that, I need to know specifically what capability I should be looking for with this, I can check out the Java code, was just wondering if there was anything documented